Divyanshu | Test | CSS Improvement: Version 3 - CSS Refactor

dvCodeWorld opened the pull request:
https://app.clickup.com/t/86d0fexwc

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- Style
  - Removed many small-screen responsive overrides; narrow-view layouts will generally follow base styles.
  - Commented/cleaned up numerous component and global style blocks; some modals, headers and inputs simplified.
  - Renamed form-field height variable; small input heights updated.
  - Updated spacing utilities (row-gap → row-gap15) across templates.

- Refactor
  - Removed legacy Subscription V1 and Add‑ons screens (UI, styles, and components).

- UI
  - Minor back-button markup update in Billing Details.
  - Removed custom tooltip styling; tooltips use default theme.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Commit:
Dilpreet | Stage | End Subscription - Mark subscription cancel after 21 days from expiry (Trial and active expired)

Commit:
fixed activate plan api issue

dvCodeWorld committed:
feat: sales person v3

Commit:
Dilpreet | Stage | Prod - attachment removed from ledger entry after download API run

ankitdubeyoffice committed:
Prod - Button to clear filter is not available on Aging report page

ankitdubeyoffice committed:
Prod - Account unique name is not going in create cash invoice flow

ankitdubeyoffice committed:
Prod - Tab key navigation fails to show focus indicator in journal entry page

Commit:
Dilpreet | Stage | Prod Bugs

dvCodeWorld committed:
Merge pull request #15713 from Walkover-Web-Solution/86d04y1zx-stage

Dilpreet | Stage | Prod - attachment removed from ledger entry after download API run

dvCodeWorld committed:
Merge pull request #15735 from Walkover-Web-Solution/86d04y1zy-stage

Dilpreet | Stage | Prod Bugs

ankitdubeyoffice committed:
use enum

ankitdubeyoffice committed:
work copy voucher

dvCodeWorld committed:
Merge pull request #15731 from Walkover-Web-Solution/86czxh0jp-stage

Ankit | Stage | Prod - Tab key navigation fails to show focus indicator in journal entry page

ankitdubeyoffice committed:
Merge pull request #15728 from Walkover-Web-Solution/86d0b3wj7-stage

Ankit | Stage |  Prod - Account unique name is not going in create cash invoice flow

ankitdubeyoffice committed:
Merge pull request #15720 from Walkover-Web-Solution/86d0a4qmr-stage

Ankit | Stage | Prod - Button to clear filter is not available on Aging report page

dvCodeWorld committed:
Merge branch 'beta-stage' into sync-material-test

dilpreetsinghofficial committed:
Merge pull request #15740 from Walkover-Web-Solution/sync-material-test

Main: Test to Beta-stage

dvCodeWorld committed:
Working on sales person V3

dvCodeWorld committed:
feat: add localization support for sales person archive and delete dialogs

dvCodeWorld committed:
feat: hide edit/delete actions for archived sales persons and update message formatting

dvCodeWorld committed:
pr changes on unit testing

dvCodeWorld committed:
pr changes on self review

dvCodeWorld committed:
refactor: remove redundant menu styling classes and consolidate column wrapper styles

dvCodeWorld committed:
Revert "Mark address default in create/update address flow"

This reverts commit 5e15b05ea0f4c9081959ebd88731d87db04fe9a1.

dvCodeWorld committed:
Revert "add default key"

This reverts commit 2149aa4224f73ef9e109456b6d180a4cce3cfd11.

dvCodeWorld committed:
Revert "Test - Show default address on get all address page in separate column"

This reverts commit cb59c2934c98fad44d47e1c4371b7b6d1621fec1.

dvCodeWorld committed:
Revert "Prod/Test -Data is not getting export with Advance filter by Inventory"

This reverts commit 88f0bf42e1afd69eb1d0b39a8e1a5c7b61abe8e5.

dvCodeWorld committed:
Revert "Dilpreet | Stage | End Subscription - Mark subscription cancel after 21 days from expiry (Trial and active expired)"

This reverts commit d4dbdaaedb168f3fa3d8f94264a62bf4d460181e.

ankitdubeyoffice committed:
Design issues

ankitdubeyoffice committed:
change

ankitdubeyoffice committed:
revert

ankitdubeyoffice committed:
revert

ankitdubeyoffice committed:
us const

ankitdubeyoffice committed:
Pagination is not working on imported transactions on ledger

ankitdubeyoffice committed:
remove

Commit:
Dilpreet | Test | Thermal Template Bugs

dvCodeWorld committed:
style: update month-year picker placeholder color and add custom class

dvCodeWorld committed:
Update style-2.scss

ankitdubeyoffice committed:
Pagination is not working on imported transactions on ledger

dilpreetsinghofficial committed:
Update build.yml

dilpreetsinghofficial committed:
Merge pull request #15757 from Walkover-Web-Solution/86d0dc9wx-test

Test TO Beta-stage

dilpreetsinghofficial committed:
Merge pull request #15763 from Walkover-Web-Solution/function-fix

Ankit | Unable to create journal entry, exception is coming

dvCodeWorld committed:
fix: update sales person transfer logic and status column header

ankitdubeyoffice committed:
Merge pull request #15760 from Walkover-Web-Solution/revert-task-stage

Revert task stage

dvCodeWorld committed:
pr changes

ankitdubeyoffice committed:
Exception is coming when selecting account for imported transaction on ledger

ankitdubeyoffice committed:
Searching not working on  OCR list page

dvCodeWorld committed:
fix: handle sales person transfer in ledger and account forms

ankitdubeyoffice committed:
conflict resolve

ankitdubeyoffice committed:
Leave page confirmation pop up coming on send mail without changing anything

ankitdubeyoffice committed:
Searching not working on  OCR list page

dvCodeWorld committed:
Merge branch 'beta-stage' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into 86d07960a-stage

dvCodeWorld committed:
sync: sales person v3 in stage manually

dvCodeWorld committed:
Merge pull request #15696 from Walkover-Web-Solution/86d07960a-stage

Divyanshu | Stage | Sales person support - Version 3 (Archive sales Person)

Commit:
Prod Bugs

dvCodeWorld committed:
Merge pull request #15795 from Walkover-Web-Solution/86d02qw5c-stage

Dilpreet | Stage | Prod Bugs

ankitdubeyoffice committed:
Email Id validation is not working on email field while sharing permission to user

ankitdubeyoffice committed:
check for tax

dvCodeWorld committed:
fix: improve sales person delete flow by handling linked entities cases

dilpreetsinghofficial committed:
Merge pull request #15801 from Walkover-Web-Solution/86d0fh8yb-stage

Divyanshu | Stage | Test - Delete sales person taking multiple steps and sending incorrect messages after delete sales person

ankitdubeyoffice committed:
check label value

dvCodeWorld committed:
Merge pull request #15804 from Walkover-Web-Solution/86d0e78c5-stage

Ankit | Stage |  Prod sanity - BUG

dvCodeWorld committed:
fix: add delete unique name and rename archive column to archiveStatus

ankitdubeyoffice committed:
check regex

dvCodeWorld committed:
Merge pull request #15806 from Walkover-Web-Solution/86d0fh8yb-v1-stage

Divyanshu | Stage | Test - Delete sales person taking multiple steps and sending incorrect messages after delete sales person

dvCodeWorld committed:
fix: remove extra plus symbol from cancel button text in voucher list

ankitdubeyoffice committed:
Merge pull request #15808 from Walkover-Web-Solution/86d0fh8yb-v1-stage

hot fix

ankitdubeyoffice committed:
check null condition

dvCodeWorld committed:
Merge pull request #15810 from Walkover-Web-Solution/86d0e78c5-stage

Ankit | check null

ankitdubeyoffice committed:
Sender's name/product's name list is not getting with all names after clear the page on receipt note page

dvCodeWorld committed:
Merge pull request #15815 from Walkover-Web-Solution/86d0eg7jp-stage

Ankit | Stage | Email Id validation is not working on email field while sharing permi…

ankitdubeyoffice committed:
Design issue

dvCodeWorld committed:
refactor: remove media queries and unused styles from component stylesheets

dvCodeWorld committed:
Merge branch 'giddh-2.0' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into 86d0fexwc-test

dvCodeWorld committed:
refactor: standardize row gap class to row-gap15 across components

ankitdubeyoffice committed:
refactor: replace hardcoded breakpoint values with BREAKPOINT_SCREEN_SIZE enum

dvCodeWorld committed:
Merge pull request #15820 from Walkover-Web-Solution/test-86d0fexwc-test

refactor: replace hardcoded breakpoint values with BREAKPOINT_SCREEN_…

ankitdubeyoffice committed:
change variable name

dvCodeWorld committed:
style: remove tooltip-black class and unused CSS styles across components

Commit:
Dilpreet | Stage | Prod Bugs

dvCodeWorld committed:
style: replace h1-h4 tags with p tags for semantic HTML structure

Commit:
fixed issue

dvCodeWorld committed:
Merge pull request #15822 from Walkover-Web-Solution/85zt28z6s-stage

Dilpreet | Stage | Prod Bugs

ankitdubeyoffice committed:
check

ankitdubeyoffice committed:
conflict resolve

dvCodeWorld committed:
Merge pull request #15821 from Walkover-Web-Solution/test-86d0fexwc-test

Ankit |  Remove Mobile Screen.

dvCodeWorld committed:
refactor: update style directory path from style to styles in asset references

dvCodeWorld committed:
Merge branch '86d0fexwc-test' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into 86d0fexwc-test

dvCodeWorld committed:
feat: implement 7-1 SCSS architecture pattern with Material Design integration

dvCodeWorld committed:
feat: add cross-browser SCSS mixins and utility classes for text, cursor and opacity

ankitdubeyoffice committed:
remove css unused

ankitdubeyoffice committed:
conflict resolve

Commit:
Dilpreet | Stage | Prod Bugs

dvCodeWorld committed:
refactor: update breakpoint constants and remove unused modal header styles

dvCodeWorld committed:
Merge branch '86d0fexwc-test' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into test-86d0fexwc-test

dvCodeWorld committed:
fix: wrong comment pattern

dvCodeWorld committed:
Merge branch 'giddh-2.0' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into 86d0fexwc-test

dvCodeWorld committed:
refactor: remove unused CSS classes and standardize delete icon styling

ankitdubeyoffice committed:
remove

ankitdubeyoffice committed:
conflict resolve

dvCodeWorld committed:
style: improve loader styling and clean up material overrides

dvCodeWorld committed:
Merge branch '86d0fexwc-test' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into test-86d0fexwc-test

dvCodeWorld committed:
Merge remote-tracking branch 'refs/remotes/origin/test-86d0fexwc-test' into test-86d0fexwc-test

dvCodeWorld committed:
Merge pull request #15823 from Walkover-Web-Solution/test-86d0fexwc-test

Ankit | Remove css unused

ankitdubeyoffice committed:
Master button is unclickable on return

ankitdubeyoffice committed:
remove

dvCodeWorld committed:
refactor: move inline styles to component CSS and clean up unused classes

dvCodeWorld committed:
style: update color variables to use Material Design tokens

Commit:
fixed bug

ankitdubeyoffice committed:
Sender's name/product's name list is not getting with all names after clear the page on receipt note page

dvCodeWorld committed:
style: update CSS class names and variable references for consistency

dvCodeWorld committed:
refactor: optimize mat-paginator placement and structure across multiple components

Commit:
fixed bug

dvCodeWorld committed:
style: update color variables from mat-primary to text/bg-primary

dilpreetsinghofficial commented on apps/web-giddh/src/app/accounting/accounting.component.scss:
```diff
@@ -132,79 +132,82 @@
+*/
 
 :host ::ng-deep .menu {
     border-left: 0 !important;
```
combine this css in single class

dvCodeWorld replied:
Removed

dilpreetsinghofficial commented on apps/web-giddh/src/app/accounting/journal-voucher/journal-voucher.component.scss:
```diff
@@ -70,67 +70,70 @@
+// }
 
 :host ::ng-deep .menu {
     box-shadow: none !important;
```
remove duplicate class

dilpreetsinghofficial commented on apps/web-giddh/src/app/accounting/journal-voucher/voucher/voucher.component.html:
```diff
@@ -367,8 +367,8 @@ <h3 class="pd-1 bd-rb primary-color" *ngIf="selectedField === 'stock'">{{ locale
     <div class="sidebar-list form-control" [hidden]="!showDiscountSidebar">
-        <div class="bd-rb d-flex justify-content-between align-items-center discount-sidebar-section">
-            <h3 class="primary-color">{{ localeData?.discount_list }}</h3>
+        <div class="border-bottom d-flex justify-content-between align-items-center pd">
```
check pd class

dvCodeWorld replied:
done

dilpreetsinghofficial commented on apps/web-giddh/src/app/accounting/journal-voucher/voucher/voucher.component.scss:
```diff
@@ -27,25 +24,25 @@ $text-overflow-ellipsis-width: 255px;
 }
 
 .hilighted {
     &>a {
```
please move this css into .hilighted class

dvCodeWorld replied:
done

dilpreetsinghofficial commented on apps/web-giddh/src/app/activity-logs/activity-logs.component.html:
```diff
@@ -279,7 +279,7 @@
                         mat-cell
                         *matCellDef="let element"
-                        class="position-relative"
+                        class="position-relative "
```
remove white space and check in overall where position-relative class are format

dilpreetsinghofficial commented on apps/web-giddh/src/app/ai-ocr/ai-ocr-list/ai-ocr-list.component.html:
```diff
@@ -99,13 +99,13 @@
                             mat-cell
                             *matCellDef="let element"
-                            class="position-relative text-left"
+                            class="position-relative  text-left"
```
remove white space

dvCodeWorld replied:
done in all

dilpreetsinghofficial commented on apps/web-giddh/src/app/ai-ocr/ai-ocr-list/ai-ocr-list.component.html:
```diff
@@ -134,10 +134,10 @@
                             ></ng-container>
                         </th>
-                        <td mat-cell *matCellDef="let element" class="position-relative text-left">
+                        <td mat-cell *matCellDef="let element" class="position-relative  text-left">
```
format code where add text-left

dilpreetsinghofficial commented on apps/web-giddh/src/app/billing-details/billing-details.component.scss:
```diff
@@ -105,7 +109,7 @@
             height: 39px;
         }
-        .red-text {
+        .text-red {
```
move into global

dilpreetsinghofficial commented on apps/web-giddh/src/app/contact/account-statement/account-statement.component.html:
```diff
@@ -248,11 +246,11 @@ <h2 class="font-30 text-right">{{ localeData?.statement_of_accounts }}</h2>
                                     mat-header-cell
                                     *matHeaderCellDef
-                                    class="bg-black text-white font-16 fw-900"
+                                    class="bg-black text-white font-size-16 fw-900"
```
make global class and use here for font weight and check in over all giddh

fw-900

dilpreetsinghofficial commented on apps/web-giddh/src/app/contact/aging-report/aging-report.component.html:
```diff
@@ -210,21 +196,21 @@ <h5>
+    <div class="icon-pointer cursor-pointer">
         <div
-            class="icon-sort-asc one text-light-2 d-block font-xxs"
+            class="icon-sort-asc one text-light d-block font-xxs"
```
make global and check in over all giddh 

font-xxs

dilpreetsinghofficial commented on apps/web-giddh/src/app/contact/aging-report/aging-report.component.html:
```diff
@@ -332,7 +318,7 @@ <h5>
                             (click)="openAgingDropDown()"
                             [matMenuTriggerFor]="agingDropdown"
-                            class="font-italic"
+                            class="font-style-font-style-italic"
```
```suggestion
                            class="font-style-italic"
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/contact/contact.component.html:
```diff
@@ -915,7 +891,7 @@ <h4>{{ account?.gstIn }}</h4>
             </ng-container>
 
-            <ng-container *ngIf="activeTab === 'vendor'" class="sticky-table">
+            <ng-container *ngIf="activeTab === 'vendor'">
```
revert sticky class

dvCodeWorld replied:
no class present with this name

dilpreetsinghofficial commented on apps/web-giddh/src/app/contact/payment-aside/payment-aside.component.html:
```diff
@@ -140,8 +140,8 @@
                     <div class="form-group mr-r15" *ngIf="isPayClicked">
-                        <label class="relative d-flex"
-                            >{{ commonLocaleData?.app_enter_otp }}<span class="red-text">*</span>
+                        <label class="position-relatived-flex"
```
```suggestion
                        <label class="position-relative d-flex"
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/contact/preview/preview.component.html:
```diff
@@ -11,8 +11,8 @@
-            <div class="d-flex align-items-center column-gap5 pd-15 w-100 list-responsive row-gap1">
-                <div class="position-relative search-field flex-grow-1">
+            <div class="d-flex align-items-center column-gap-05 pd-15 w-100 list-responsive row-gap-1">
+                <div class="position-relative  search-field flex-grow-1">
```
format code

dilpreetsinghofficial commented on apps/web-giddh/src/app/contact/preview/preview.component.html:
```diff
@@ -63,8 +63,8 @@
         <div class="preview-content">
-            <div class="relative overflow-visible">
-                <div class="clearfix overflow-visible">
+            <div class="position-relativeoverflow-visible">
```
```suggestion
            <div class="position-relative overflow-visible">
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/custom-fields/create-edit/create-edit.component.html:
```diff
@@ -8,15 +8,15 @@
     <giddh-page-loader *ngIf="showLoader"></giddh-page-loader>
     <ng-container *ngIf="!showLoader">
-        <div class="relative create-custom-field-page mr-t2">
+        <div class="position-relativecreate-custom-field-page mr-t2">
```
```suggestion
        <div class="position-relative create-custom-field-page mr-t2">
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/downloads/components/imports/imports.component.html:
```diff
@@ -15,9 +15,9 @@
+                        <div class="clear-both">
                             <div class="col-xs-12 col-sm-12">
-                                <div class="d-inline-block pull-left mr-2 middle eway-bill-date">
+                                <div class="d-inline-block pull-left mr-2 vertical-align-middleeway-bill-date">
```
```suggestion
                                <div class="d-inline-block pull-left mr-2 vertical-align-middle eway-bill-date">
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/downloads/downloads.component.html:
```diff
@@ -5,8 +5,8 @@
     <hamburger-menu class="hamburger-menu"></hamburger-menu>
-    <div class="relative overflow-visible">
-        <div class="clearfix petty-cash overflow-visible">
+    <div class="position-relativeoverflow-visible">
```
```suggestion
    <div class="position-relative overflow-visible">
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/expenses/components/pending-list/pending-list.component.html:
```diff
@@ -113,7 +113,7 @@
                         *ngIf="expenses.fileNames"
                         (click)="openZoomImageView($event, expenses.fileNames)"
-                        class="icon-image middle text-gray"
+                        class="icon-image vertical-align-middletext-light"
```
```suggestion
                        class="icon-image vertical-align-middle text-light"
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/expenses/components/rejected-list/rejected-list.component.html:
```diff
@@ -111,7 +111,7 @@
                         *ngIf="rejected.fileNames"
                         (click)="openZoomImageView(rejected.fileNames)"
-                        class="icon-image middle text-gray pointer"
+                        class="icon-image vertical-align-middletext-light pointer"
```
```suggestion
                        class="icon-image vertical-align-middle text-light pointer"
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/expenses/expenses.component.html:
```diff
@@ -4,9 +4,9 @@
     (commonLocaleData)="commonLocaleData = $event"
 >
-    <div class="relative overflow-visible">
+    <div class="position-relativeoverflow-visible">
```
```suggestion
    <div class="position-relative overflow-visible">
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/financial-reports/components/grid-row/grid-row.component.html:
```diff
@@ -85,7 +85,7 @@
     >
         <div class="row row-item">
-            <div class="col-4 account no-select d-flex align-items-center row-height" [style.padding-left]="padding + 20 + 'px !important'">
+            <div class="col-4 account user-select-n d-flex align-items-center row-height" [style.padding-left]="padding + 20 + 'px !important'">
```
user-select-n ???

dilpreetsinghofficial commented on apps/web-giddh/src/app/gst/filing/header/filing-header.component.html:
```diff
@@ -1,29 +1,29 @@
 
-            <div class="text-middle nav-top">
-                <div class="text-middle head head-font-weight d-flex align-items-center column-gap1">
+            <div class="text-vertical-align-middlenav-top">
```
```suggestion
            <div class="text-vertical-align-middle nav-top">
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/gst/filing/header/filing-header.component.html:
```diff
@@ -1,29 +1,29 @@
-            <div class="text-middle nav-top">
-                <div class="text-middle head head-font-weight d-flex align-items-center column-gap1">
+            <div class="text-vertical-align-middlenav-top">
+                <div class="text-vertical-align-middlehead head-font-weight d-flex align-items-center column-gap-1">
```
please check text-vertical-align-middlehead is concat

dilpreetsinghofficial commented on apps/web-giddh/src/app/gst/filing/tabs/push-to-gstin/components/b2cs-summary/b2cs-summary.component.html:
```diff
@@ -8,44 +8,44 @@ <h4 class="clearfix gstr-summary-title">{{ localeData?.filing?.b2c_small_details
 
                         <th class="text-right">
-                            <div class="position-relative">
+                            <div class="position-relative ">
```
format code

dilpreetsinghofficial commented on apps/web-giddh/src/app/gst/filing/tabs/push-to-gstin/components/nil-summary/nil-summary.component.html:
```diff
@@ -1,28 +1,28 @@
                         </th>
                         <th class="text-right">
-                            <div class="position-relative">
+                            <div class="position-relative ">
```
remove white space

dilpreetsinghofficial commented on apps/web-giddh/src/app/gst/filing/tabs/reconcilation/reconcilation.component.html:
```diff
@@ -242,7 +238,7 @@
     <tr>
         <th class="text-left width-vendor" *ngIf="selectedGst === 'gstr1'">
-            <div class="position-relative">
+            <div class="position-relative ">
```
check white space in over all file

dilpreetsinghofficial commented on apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.html:
```diff
@@ -62,7 +62,7 @@
                                                     let i = index
                                                 "
-                                                class="position-relative"
+                                                class="position-relative "
```
format code

dilpreetsinghofficial commented on apps/web-giddh/src/app/gst/gst.component.html:
```diff
@@ -331,7 +331,7 @@
                                 ></path>
                             </svg>
-                            <i class="text-middle p-0 mr-l05">{{ localeData?.gstr2_push_disabled }}</i>
+                            <i class="text-vertical-align-middlep-0 mr-l05">{{ localeData?.gstr2_push_disabled }}</i>
```
```suggestion
                            <i class="text-vertical-align-middle p-0 mr-l05">{{ localeData?.gstr2_push_disabled }}</i>
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/home/components/ratio-analysis/ratio-analysis-chart.component.html:
```diff
@@ -14,7 +14,7 @@
         </div>
 
-        <mat-card-content class="position-relative ratio-analysis-panel-body">
+        <mat-card-content class="position-relative  ratio-analysis-panel-body">
```
format code

dilpreetsinghofficial commented on apps/web-giddh/src/app/home/components/total-overdues/total-overdues-chart.component.html:
```diff
@@ -30,7 +30,7 @@
         </div>
 
-        <mat-card-content class="position-relative">
+        <mat-card-content class="position-relative ">
```
check postiion relative space in over all giddh

dilpreetsinghofficial commented on apps/web-giddh/src/app/import-excel/upload-file/upload-file.component.html:
```diff
@@ -29,7 +29,7 @@ <h2 class="text-center">{{ localeData?.import }}
         </div>
     </div>
-    <div class="upload-excel max-width-500 center-block box mr-t2 dashed-border">
+    <div class="upload-excel max-width-500 center-blockpd-15mr-t2 dashed-border">
```
```suggestion
    <div class="upload-excel max-width-500 center-block pd-15 mr-t2 dashed-border">
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/inventory/components/aside-transfer-pane/aside-transfer-pane.component.html:
```diff
@@ -49,7 +49,7 @@ <h3>Create</h3>
         </ng-container>
         <ng-container *ngSwitchDefault>
-            <div class="aside-body flexy-child vertical-align-middle aside-panel-ledger" vr-item>
+            <div class="aside-body flexy-child vertical-align-vertical-align-middleaside-panel-ledger" vr-item>
```
```suggestion
            <div class="aside-body flexy-child vertical-align-vertical-align-middleaside-panel-ledger" vr-item>
            
            please check this 
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.html:
```diff
@@ -555,7 +555,7 @@ <h3>{{ activeCompany?.baseCurrencySymbol }} {{ overallTotal | giddhCurrency }}</
                                                 (selectedOption)="selectProduct($event, product)"
                                             ></reactive-dropdown-field>
-                                            <div class="position-relative wrap-hsn-box">
+                                            <div class="position-relative  wrap-hsn-box">
```
```suggestion
                                            <div class="position-relative wrap-hsn-box">
                                            
                                            remove white space in over all file
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.html:
```diff
@@ -1196,7 +1196,7 @@ <h3 class="mr-b1">Note</h3>
                                 *ngIf="isBranch || isCompanyWithSingleBranch"
                                 type="button"
-                                mat-raised-button color="primary" class="d-inline-block v-align-middle {{
+                                mat-raised-button color="primary" class="d-inline-block vertical-align-vertical-align-middle{{
```
check this

dilpreetsinghofficial commented on apps/web-giddh/src/app/invoice/preview/models/bulkUpdateModal/invoiceBulkUpdateModal.component.html:
```diff
@@ -60,7 +60,7 @@
                         </div>
 
-                        <div class="relative form-group" *ngIf="selectedField === 'dueDate'">
+                        <div class="position-relativeform-group" *ngIf="selectedField === 'dueDate'">
```
```suggestion
                        <div class="position-relative form-group" *ngIf="selectedField === 'dueDate'">
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/invoice/preview/models/bulkUpdateModal/invoiceBulkUpdateModal.component.html:
```diff
@@ -144,7 +144,7 @@
                     <div *ngIf="selectedField === 'customFields'" class="row mb-2">
                         <div class="col-md-4">
-                            <div class="relative form-group">
+                            <div class="position-relativeform-group">
```
please correct in over all file

dilpreetsinghofficial commented on apps/web-giddh/src/app/ledger/components/new-ledger-entry-panel/new-ledger-entry-panel.component.html:
```diff
@@ -386,11 +386,11 @@
                                 <span class="d-flex mr-b05">
-                                    <span class="align-item-center pd-r1">{{ blankLedger.otherTaxModal?.appliedOtherTax?.name }}</span>
-                                    <span class="d-flex align-item-center">
+                                    <span class="d-flex flex-column justify-content-center vertical-align-middle align-items-center pd-r1">{{ blankLedger.otherTaxModal?.appliedOtherTax?.name }}</span>
```
check in over all file

dilpreetsinghofficial commented on apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.html:
```diff
@@ -464,7 +459,7 @@ <h1>{{ localeData?.no_transaction_found_message }}</h1>
                                                         [attr.data-title]="commonLocaleData?.app_particular"
                                                         [id]="'credit-id-' + i"
-                                                        class="position-relative col-9 select2-parent select2-ledger d-flex align-items-center justify-content-between column-gap1"
+                                                        class="position-relative  col-9 select2-parent select2-ledger d-flex align-items-center justify-content-between column-gap-1"
```
remove white space

dilpreetsinghofficial commented on apps/web-giddh/src/app/ledger/components/update-ledger-tax-control/update-ledger-tax-control.component.html:
```diff
@@ -1,4 +1,4 @@
-<span class="position-relative custom-dropdown-ledger">
+<span class="position-relative  custom-dropdown-ledger">
```
remove white space

dilpreetsinghofficial commented on apps/web-giddh/src/app/ledger/ledger.component.html:
```diff
@@ -2282,14 +2255,13 @@ <h3 class="ledger-main-head">
                                 <tbody>
                                     <tr class="total-col" *ngIf="ledgerTxnBalance?.closingBalance?.type === transactionType.Credit">
-                                        <td colspan="12" class="col-xs-12 text-right white h32 pd-r1">
+                                        <td colspan="12" class="col-xs-12 text-right text-whiteh32 pd-r1">
```
```suggestion
                                        <td colspan="12" class="col-xs-12 text-right text-white h32 pd-r1">
```

dilpreetsinghofficial commented on apps/web-giddh/src/app/ledger/ledger.component.html:
```diff
@@ -2164,7 +2137,7 @@ <h3 class="ledger-main-head">
                                                     </mat-grid-tile>
                                                     <mat-grid-tile
-                                                        class="position-relative no-right-border"
+                                                        class="position-relative  no-right-border"
```
remove white space

dilpreetsinghofficial commented on apps/web-giddh/src/app/manufacturing/edit/mf.edit.component.html:
```diff
@@ -352,10 +352,10 @@ <h1 class="section-head">Source(Consumption)</h1>
                                     </td>
                                     <td>
-                                        <div class="position-relative">
+                                        <div class="position-relative ">
```
remove white space

dilpreetsinghofficial commented on apps/web-giddh/src/app/multi-currency-reports/grid-row/grid-report-row.component.html:
```diff
@@ -72,9 +72,9 @@
     >
         <div class="row row-item">
-            <div class="col-4 account no-select" [style.padding-left]="padding + 20 + 'px !important'">
+            <div class="col-4 account user-select-n" [style.padding-left]="padding + 20 + 'px !important'">
```
chekc in over all giddh

dilpreetsinghofficial commented on apps/web-giddh/src/app/import-excel/import-excel.component.scss:
```diff
@@ -103,14 +119,19 @@
     transform: scale(1.01);
 }
 
+/* UNUSED COMPONENT CLASS: lead - Auto-commented by script
```
remove commented code after testing

dvCodeWorld replied:
Sure !

dilpreetsinghofficial commented on apps/web-giddh/src/app/shared/bottom-space/bottom-space.component.ts:
```diff
@@ -0,0 +1,12 @@
+import { Component, ViewEncapsulation } from '@angular/core';
```
use case?

dvCodeWorld replied:
Used in future for bottom space in each page as discussed

dilpreetsinghofficial commented on apps/web-giddh/src/app/shared/tax-sidebar/tax-sidebar.component.ts:
```diff
@@ -189,7 +187,7 @@ export class TaxSidebarComponent implements OnInit, OnDestroy {
     * @memberof TaxSidebarComponent
     */
     public closeAsidePaneIfMobile(event?): void {
-        if (this.isMobileScreen && event && event.target.className !== "icon-bar") {
```
remove this if its unused

dvCodeWorld replied:
done

dilpreetsinghofficial commented on apps/web-giddh/src/app/ledger/components/update-ledger-discount/update-ledger-discount.component.html:
```diff
@@ -20,7 +20,7 @@
         <div class="d-flex flex-column px-3 pt-2 discount-wrapper" (click)="$event.stopPropagation()">
             <div class="value-wrapper">
                 <div class="d-flex align-items-center mr-b1">
-                    <div class="position-relative percent-wrapper custom-mat-form-field">
```
remove white space

dvCodeWorld replied:
where white space

dilpreetsinghofficial commented on apps/web-giddh/src/app/multi-currency-reports/trial-balance/components/trial-balance-grid/trial-balance-report-grid.component.html:
```diff
@@ -42,7 +42,7 @@
                                 ><strong>{{ localeData?.particular }}</strong></span
                             >
-                            <i class="position-relative font-14 pd-r1 cursor-pointer d-flex align-items-center light-gray" id="showSearch" (click)="toggleSearch()"></i>
+                            <i class="position-relative  font-size-14 pd-r1 cursor-pointer d-flex align-items-center light-gray" id="showSearch" (click)="toggleSearch()"></i>
```
format code

dvCodeWorld replied:
done

dilpreetsinghofficial commented on apps/web-giddh/src/app/new-inventory/component/branch-transfer/create-branch-transfer/create-branch-transfer.component.html:
```diff
@@ -377,7 +377,7 @@ <h2>
                                     <div class="col-lg-6">
                                         <div
-                                            class="position-relative info-field mr-b15"
+                                            class="position-relative  info-field mr-b15"
```
remove white space

dilpreetsinghofficial commented on apps/web-giddh/src/app/new-inventory/component/branch-transfer/create-branch-transfer/create-branch-transfer.component.html:
```diff
@@ -1519,7 +1519,7 @@ <h3>{{ activeCompany?.baseCurrencySymbol }} {{ overallTotal | giddhCurrency }}</
                                                         <div
                                                             *ngIf="activeIndx === i"
-                                                            class="position-relative w-100 fields-gap"
+                                                            class="position-relative  w-100 fields-gap"
```
format code

dilpreetsinghofficial commented on apps/web-giddh/src/app/new-inventory/component/branch-transfer/create-branch-transfer/create-branch-transfer.component.html:
```diff
@@ -2342,7 +2342,7 @@ <h3 class="mr-b15">{{ commonLocaleData?.app_transport_details }}</h3>
                                     color="primary"
                                     *ngIf="isBranch || isCompanyWithSingleBranch"
-                                    class="d-inline-block v-align-middle {{ branchTransferCreateEditForm.invalid }}-{{
+                                    class="d-inline-block vertical-align-vertical-align-middle{{ branchTransferCreateEditForm.invalid }}-{{
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/new-inventory/component/custom-price/customer-wise/customer-wise.component.html:
```diff
@@ -215,7 +215,7 @@
                                                 <ng-container [formGroupName]="i">
                                                     <div>
-                                                        <table class="table mb-0 no-border">
+                                                        <table class="">
```
remove this blank class

dvCodeWorld replied:
done in overall class=""

dilpreetsinghofficial commented on apps/web-giddh/src/app/new-inventory/component/manufacturing/list-manufacturing/list-manufacturing.component.html:
```diff
@@ -188,7 +188,7 @@
             </table>
         </section>
-        <div class="mt-4" *ngIf="(totalItems > pageSizeOptions[0]) && !isReportLoading">
+        <div *ngIf="(totalItems > pageSizeOptions[0]) && !isReportLoading">
```
please remove div use ngIf in mat-paginator

dilpreetsinghofficial commented on apps/web-giddh/src/app/new-inventory/component/stock-create-edit/stock-create-edit.component.html:
```diff
@@ -7,7 +7,7 @@
     <giddh-page-loader *ngIf="showLoader"></giddh-page-loader>
     <div [style.display]="showLoader ? 'none' : 'block'">
-        <div class="relative create-new-inventory-page static-tabs-on-page">
+        <div class="position-relativecreate-new-inventory-page static-tabs-on-page">
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/new-inventory/new-inventory.component.html:
```diff
@@ -3,7 +3,7 @@
 <hamburger-menu></hamburger-menu>
 
-<div class="relative new-inventory-page overflow-visible">
+<div class="position-relativenew-inventory-page overflow-visible">
```
check

dilpreetsinghofficial commented on apps/web-giddh/src/app/reports/components/purchase-register-component/purchase.register.component.html:
```diff
@@ -6,9 +6,9 @@
+        <hamburger-menu></hamburger-menu>
         <div class="report-breadcrum d-md-none d-lg-none">
-            <ul class="list-inline d-inline-block middle nav-report">
+            <ul class="list-inline d-inline-block vertical-align-middlenav-report">
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.html:
```diff
@@ -56,7 +56,7 @@
                     </button>
                 </div>
-                <div class="dis middle top-bar-content-right-side">
+                <div class="dis vertical-align-middletop-bar-content-right-side">
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/reports/components/report-details-components/report.details.component.html:
```diff
@@ -6,9 +6,9 @@
+        <hamburger-menu></hamburger-menu>
         <div class="report-breadcrum d-md-none d-lg-none">
-            <ul class="list-inline d-inline-block middle nav-report">
+            <ul class="list-inline d-inline-block vertical-align-middlenav-report">
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/reports/components/sales-register-expand-component/sales.register.expand.component.html:
```diff
@@ -56,7 +56,7 @@
                     </button>
                 </div>
-                <div class="dis middle top-bar-content-right-side">
+                <div class="dis vertical-align-middletop-bar-content-right-side">
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/sales/discount-list/discountList.component.html:
```diff
@@ -25,7 +25,7 @@
                 <label>Percent</label>
 
-                <div class="position-relative percent-wrapper">
+                <div class="position-relative  percent-wrapper">
```
format code

dilpreetsinghofficial commented on apps/web-giddh/src/app/search/components/search-filter/search-filter.component.html:
```diff
@@ -98,7 +98,7 @@
                             </span>
-                            <span class="position-relative form-group  option-value-field" *ngIf="item?.value.queryType === 'closingBalance'">
+                            <span class="position-relative  form-group  option-value-field" *ngIf="item?.value.queryType === 'closingBalance'">
                                 <select-field
```
remove white spaces

dilpreetsinghofficial commented on apps/web-giddh/src/app/settings/discount/discount.component.html:
```diff
@@ -5,7 +5,7 @@
     (translationComplete)="translationComplete($event)"
 >
-    <div class="discount box clearfix section-center position-relative overflow-visible">
+    <div class="discountpd-15clearfix section-center position-relative overflow-visible">
```
```suggestion
    <div class="discount pd-15 clear-both section-center position-relative overflow-visible">
```

dvCodeWorld replied:
done in over all

dilpreetsinghofficial commented on apps/web-giddh/src/app/settings/financial-year/financial-year.component.html:
```diff
@@ -5,16 +5,16 @@
     (translationComplete)="translationComplete($event)"
 >
-    <div class="section-center box clearfix overflow-visible">
+    <div class="section-centerpd-15clearfix overflow-visible">
```
check this

dilpreetsinghofficial commented on apps/web-giddh/src/app/settings/triggers-old/triggers.component.html:
```diff
@@ -8,7 +8,7 @@
         <giddh-page-loader></giddh-page-loader>
     </div>
-    <div class="discount box clearfix section-center position-relative overflow-visible" *ngIf="!isShowIntegrated && !communicationPlatforms?.MSG91?.isConnected && !isCommunicationPlatformsLoading">
+    <div class="discountpd-15clearfix section-center position-relative overflow-visible" *ngIf="!isShowIntegrated && !communicationPlatforms?.MSG91?.isConnected && !isCommunicationPlatformsLoading">
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/shared/bank-integration/bank-integration.component.html:
```diff
@@ -5,7 +5,7 @@
 >
     <div *ngIf="!isLoading && connectedBankAccounts?.length === 0">
-        <div class="integration box clearfix section-center position-relative overflow-visible">
+        <div class="integrationpd-15clearfix section-center position-relative overflow-visible">
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/shared/bank-integration/bank-integration.component.html:
```diff
@@ -27,7 +27,7 @@
             "
         >
-            <div class="clearfix box mat-tab-bottom-spacing">
+            <div class="clearfixpd-15mat-tab-bottom-spacing">
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/shared/header/components/master/master.component.html:
```diff
@@ -73,7 +73,7 @@
                                     id="subGrp_{{ item.name }}"
                                     nav-vr-item
-                                    class="list-item d-block "
+                                    class="list-item isAcc d-block "
```
remove unused

dilpreetsinghofficial commented on apps/web-giddh/src/app/shared/header/components/master/master.component.html:
```diff
@@ -106,7 +106,7 @@
                                 <a
                                     href="javascript:void(0);"
-                                    class="list-item isAcc d-block"
+                                    class="list-item  d-block"
```
remove white space

dilpreetsinghofficial commented on apps/web-giddh/src/app/shared/ledger-statement-t-view/ledger-statement.component.html:
```diff
@@ -449,14 +445,13 @@
                                 <tbody>
                                     <tr class="total-col" *ngIf="ledgerTxnBalance?.closingBalance?.type === 'CREDIT'">
-                                        <td colspan="12" class="col-xs-12 text-right white h32 pd-r1">
+                                        <td colspan="12" class="col-xs-12 text-right text-whiteh32 pd-r1">
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/shared/purchase-order-preview/purchase-order-preview.component.html:
```diff
@@ -13,7 +13,7 @@
         <div class="row mr-b1">
             <div class="col-sm-12 pdf-viewer">
-                <div class="position-relative" *ngIf="!pdfPreviewHasError && !pdfPreviewLoaded">
+                <div class="position-relative " *ngIf="!pdfPreviewHasError && !pdfPreviewLoaded">
```
remove white space in overall

dilpreetsinghofficial commented on apps/web-giddh/src/app/shared/triggers/components/advance-trigger/advance-trigger.component.html:
```diff
@@ -19,7 +19,7 @@
         </button>
         <div
-            class="discount box clearfix section-center position-relative overflow-visible"
+            class="discountpd-15clearfix section-center position-relative overflow-visible"
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html:
```diff
@@ -1056,7 +1056,7 @@ <h2 class="font-16 mb-3 strong d-flex align-items-center plan-summary-heading">
                             </p>
                         </div>
-                        <div class="position-relative mt-3" *ngIf="firstStepForm.get('planUniqueName').value">
+                        <div class="position-relative  mt-3" *ngIf="firstStepForm.get('planUniqueName').value">
```
formast code

dilpreetsinghofficial commented on apps/web-giddh/src/app/subscription/move-company/move-company.component.html:
```diff
@@ -12,11 +12,11 @@
             <div class="row mb-2">
                 <div class="move-wrapper" [ngClass]="subscriptionMove ? 'col-9' : 'col-6'">
-                    <div *ngIf="subscriptionMove" class="position-relative move-select">
+                    <div *ngIf="subscriptionMove" class="position-relative  move-select">
```
format code

dilpreetsinghofficial commented on apps/web-giddh/src/app/theme/ngx-date-range-picker/ngx-daterangepicker.component.html:
```diff
@@ -116,7 +116,7 @@
                             <div class="wrap-selected-date">
                                 <div
-                                    class="relative date-container-one"
+                                    class="position-relativedate-container-one"
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/theme/tax-authority/reports/account-wise-report/account-wise-report.component.html:
```diff
@@ -3,7 +3,7 @@
 
     <!-- Tax Sidebar code  -->
-    <div class="relative w-100">
+    <div class="position-relativew-100">
```
correct this

dilpreetsinghofficial commented on apps/web-giddh/src/app/theme/tax-authority/reports/rate-wise-report/rate-wise-report.component.html:
```diff
@@ -3,7 +3,7 @@
 
     <!-- Tax Sidebar code  -->
-    <div class="relative w-100">
+    <div class="position-relativew-100">
```
correct this

dvCodeWorld committed:
style: standardize CSS class names and remove unused styles

dvCodeWorld committed:
style: update CSS class names and remove unused styles

dvCodeWorld committed:
Merge branch 'giddh-2.0' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into 86d0fexwc-test

dvCodeWorld committed:
format style

dvCodeWorld committed:
refactor: consolidate styles into main.scss and clean up unused CSS classes

dvCodeWorld committed:
style: update color variable names from legacy to theme-based naming

dvCodeWorld committed:
style: update UI layout and button styles across multiple components

dvCodeWorld committed:
fix: update CSS classes for dark theme and improve UI spacing

dvCodeWorld committed:
Merge branch 'giddh-2.0' into 86d0fexwc-test

Commit:
fixed bug + PR changes

dilpreetsinghofficial committed:
Merge pull request #15824 from Walkover-Web-Solution/87d02qw4c-stages

Dilpreet | Stage | Prod Bugs

dvCodeWorld committed:
style: update theme colors and layout for dark mode compatibility

ankitdubeyoffice committed:
Merge pull request #15812 from Walkover-Web-Solution/86d0e78c5-stage

Ankit | Stage |  Prod - Searching not working on OCR list page

ankitdubeyoffice committed:
Merge pull request #15832 from Walkover-Web-Solution/86d0d4m29-stage

Ankit | stage | Sender's name/product's name list is not getting with all names after…

ankitdubeyoffice committed:
Merge pull request #15826 from Walkover-Web-Solution/86d0cch1y-stage

Ankit | Stage | Master button is unclickable on return

dvCodeWorld committed:
fix: remove overflow-visible class from invoice list container

ankitdubeyoffice committed:
Support of detailed VAT report

ankitdubeyoffice committed:
check null

ankitdubeyoffice committed:
add bread crumb

ankitdubeyoffice committed:
separate service for bread crum

dvCodeWorld committed:
feat: add letter spacing utilities and apply to Material typography config

dvCodeWorld committed:
Merge branch 'giddh-2.0' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into 86d0fexwc-test

dvCodeWorld committed:
style: remove unused CSS styles in expenses component

Commit:
Test to Stage

Commit:
self review

Commit:
fixed duplicate method issue

dvCodeWorld committed:
bug fixes

Commit:
conflict resolver

dilpreetsinghofficial committed:
Merge pull request #15872 from Walkover-Web-Solution/97fvcfjux-stage

Test to Stage

dilpreetsinghofficial committed:
Revert "Ankit | Test | Version 6 - Support of detailed VAT report"

dvCodeWorld committed:
Merge pull request #15878 from Walkover-Web-Solution/revert-15856-85ztm6e8u-test

Revert "Ankit | Test | Version 6 - Support of detailed VAT report"

dvCodeWorld committed:
style: improve UI alignment and spacing in contact and template components

Commit:
fixed bug

ankitdubeyoffice committed:
Test - Incorrect tax number label displaying in UK vat detailed report

Commit:
Dilpreet | Stage | Update postbuild for UK region

ankitdubeyoffice committed:
Merge pull request #15892 from Walkover-Web-Solution/86d0fz4yz-stage

Dilpreet | Stage | Update postbuild for UK region

Commit:
Dilpreet | Stage | Prod - Continuous loader appears when the user is redirected from the payment subscription page.

Commit:
Dilpreet | Stage | After login redirect user to the page he visited through URL

Commit:
fixed root route issue

ankitdubeyoffice committed:
Expand button is not working on Trial/balace /P&L report

dilpreetsinghofficial committed:
Merge pull request #15903 from Walkover-Web-Solution/86d0nbn55-test

Expand button is not working on Trial/balace /P&L report

Commit:
conitues loop issue on signup

ankitdubeyoffice committed:
use common for no data

dilpreetsinghofficial committed:
Merge pull request #15909 from Walkover-Web-Solution/86d0nbn55-test-v1

Ankit | stage - PDF preview page is distorted

dilpreetsinghofficial committed:
Merge pull request #15599 from Walkover-Web-Solution/86cytt8ne-stages

Dilpreet | Stage | End Subscription - Mark subscription cancel after 21 days from expiry (Trial and active expired)

dvCodeWorld committed:
Merge branch 'giddh-2.0' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into 86d0fexwc-test

Commit:
Dilpreet | Stage | OCR V2 Revert from Stage

dilpreetsinghofficial committed:
Merge pull request #15911 from Walkover-Web-Solution/85zrgbztx-stages

Dilpreet | Stage | OCR V2 Revert from Stage

dvCodeWorld committed:
Merge pull request #15898 from Walkover-Web-Solution/67fvcfjuz-stage

Dilpreet | Stage | Prod - Continuous loader appears when the user is redirected from the payment subscription page.

dvCodeWorld committed:
style: change no data message tags from p to h2 for better accessibility

dvCodeWorld committed:
style: update preview component styles and fix button positioning in custom units

ankitdubeyoffice committed:
change name of column

ankitdubeyoffice committed:
change name

ankitdubeyoffice committed:
remove

dilpreetsinghofficial committed:
Merge pull request #15855 from Walkover-Web-Solution/85ztm6e8u-stage

Ankit | Stage | Version 6 - Support of detailed VAT report

dvCodeWorld committed:
fix: add initial-height class to mat-form-field elements for consistent styling

dvCodeWorld committed:
Merge pull request #15900 from Walkover-Web-Solution/86d0pcacq-stage

Dilpreet | Stage | After login redirect user to the page he visited t…hrough URL

dvCodeWorld committed:
style: adjust layout spacing and alignment in imports and expenses components

dvCodeWorld committed:
style: update UI colors and layout for better theme consistency

dvCodeWorld committed:
style: update theme colors and remove hardcoded white backgrounds

dvCodeWorld committed:
style: fix dark theme design issues

dvCodeWorld committed:
refactor: standardize no-data UI and remove unused CSS classes

dvCodeWorld committed:
style: update icon colors and adjust dark theme palette shades for better contrast

dvCodeWorld committed:
style: uncomment total-box styling in accounting components

dvCodeWorld committed:
feat: improve UI styling and layout consistency

- Added min-width-100 class to voucher date field for better field sizing
- Updated info icon color to use theme-foreground-color for consistent branding
- Added initial-height class to select multiple fields component for proper form layout
- Fixed CSS formatting and removed duplicate styles in _global.scss
- Enhanced ledger head styling with proper spacing and font weight
- Standardized button and form field styles for better visual consistency

dvCodeWorld committed:
feat: add theme foreground color to search dialogs

- Added theme-foreground-color class to daybook advance search form for consistent theming
- Added theme-foreground-color class to inventory stock report advance search dialog
- Added theme-foreground-color class to ledger advance search modal body
- Updated dialog-body class in _dialog.scss to use theme foreground color variable for text

dvCodeWorld committed:
feat: improve UI styling and theme consistency

- Updated dialog component styling to ensure consistent background colors and padding
- Added proper alignment and spacing for file attachment labels and form fields
- Removed redundant theme-specific color declarations and consolidated them using mixins
- Fixed table width and margin styling in ledger statistics section
- Enhanced password field styling with border-bottom class and mat-form-field-clean
- Optimized theme mixin to target theme classes

dvCodeWorld committed:
feat: update theme colors and UI elements for consistency

- Replaced text-primary-color with theme-foreground-color for better theme consistency across components
- Changed anchor tags to span elements in revenue chart frequency selector
- Removed unused color styles and commented out redundant theme color declarations
- Added padding utility classes for better spacing in various components
- Updated filter band and datepicker container styling for improved layout

The changes primarily focus on standardizing the color scheme

dvCodeWorld committed:
Merge pull request #15943 from Walkover-Web-Solution/86d0ur1tf-v1-86d0fexwc-test

Divyanshu | 86d0fexwc-test | CSS Version 4: Remove Plain Text (non-clickable) blue color

dvCodeWorld committed:
Merge branch 'beta-stage' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into 86d0fexwc-test

dvCodeWorld committed:
Merge branch 'giddh-2.0' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into 86d0fexwc-test

dvCodeWorld committed:
chore: remove test environment deployment from build workflow

dvCodeWorld committed:
refactor: comment out dark theme text primary color override

dvCodeWorld committed:
refactor: remove sticky header background color in ledger statement

dvCodeWorld committed:
Merge branch 'giddh-2.0' into 86d0fexwc-test
